homeassistant/components/switch/flux.py

```diff
@@ -159,8 +159,7 @@ def flux_update(self, now=None):
         """Update all the lights using flux."""
         if now is None:
             now = dt_now()
-        sunset = next_setting(self.hass, SUN).replace(
```

Comment:
This is a good example of the kind of logic we want to eliminate. Taking whatever sunset you get, then picking a new date is exactly the kind of logic that leads us to double triggers and other strange bugs.